core/templates/dev/head/pages/exploration_player/AnswerClassificationServiceSpec.js

```diff
@@ -49,17 +50,14 @@ describe('Answer classification service with string classifier disabled',
       }],
       interaction: {
         id: 'RuleTest',
-        answer_groups: [{
-          outcome: 'outcome 1',
-          rule_specs: [{
+        answer_groups: [
+          agf.create([{
             inputs: {
               x: 10
             },
             rule_type: 'Equals'
-          }]
-        }, {
-          outcome: 'outcome 2',
-          rule_specs: [{
+          }], 'outcome 1'),
```

Comment:
Seems wrong that the outcomes are strings in this test, maybe we should fix that.